### DISCUSSION TITLE
posts/local-dns-masq/

### DISCUSSION THREAD

**Discussion by giscus** [yunnysunny/yunnysunny.github.io#9] [General]
# posts/local-dns-masq/

一般公司内部都会有内部 DNS 服务器，由于托管公司内网域名，且会当成公司内部上网的专用 DNS，同时承载外网的域名解析工作。但是有时候公司内网的 DNS 服务不稳定时，就比较尴尬，不用吧，解析不了内网域名，用吧，上不了外网。这时候用 DNS 的分流工具就是解决这个问题的好办法。这篇文章会拿 win10 举例，讲解 coredns 的简单配置，来解决 DNS 分流问题。

https://blog.whyun.com/posts/local-dns-masq/

**Comment by yunnysunny** [comment]
解决烦人的DNS解析问题
